### ISSUE TITLE
formParams and fileUploads doesn't have any effect

### ISSUE THREAD

**Issue by zjor** [javalin/javalin-openapi#36]
I have the following definition:
```java
@OpenApi(
            path = "/api/v1.0/sendImage",
            methods = HttpMethod.POST,
            summary = "Sends an image to MQTT",
            security = @OpenApiSecurity(name = "basicAuth"),
            formParams = {
                    @OpenApiFormParam(name = "topic", required = true),
            },
            fileUploads = {
                    @OpenApiFileUpload(name = "image")
            }
    )
    public void sendImageToMyTopic(@NotNull Context ctx) throws Exception {
      ...
    }
```
but openApi JSON doesn't include any fields and I can't invoke my endpoint via swagger UI.
This is the JSON I get for this endpoint:

```json
...
  "/api/v1.0/sendImage": {
    "post": {
      "tags": [],
      "summary": "Sends an image to MQTT",
      "parameters": [],
      "responses": {},
      "deprecated": false,
      "security": [
        {
          "basicAuth": []
        }
      ]
    }
}
```

**Comment by dzikoysk** [comment]
I'll try to take a look on that tomorrow :)

**Comment by dzikoysk** [comment]
Sorry, I was on vacation and totally I've totally forgot about this. Looks like I didn't implement those properties in generator at all!

**Comment by dzikoysk** [comment]
Oops looks like I've implemented it for OpenApi 2.0, not 3.0.

**Comment by dzikoysk** [comment]
Regardless OpenApi 3.0 specification, upload files are a part of `requestBody` now and there are some variants to configure them:

* https://swagger.io/docs/specification/describing-request-body/file-upload

Because of that I decided to make some breaking changes and move its declaration to `requestBody` as well. Here's the example from `JavalinTest` class:

```java
requestBody = @OpenApiRequestBody(
        content = {
                @OpenApiContent(from = String.class), // simple type
                @OpenApiContent(from = EntityDto[].class), // array
                @OpenApiContent(from = LombokEntity.class), // lombok
                @OpenApiContent(mimeType = "image/png", type = "string", format = "base64"), // single file upload,
                @OpenApiContent(mimeType = "multipart/form-data", properties = {
                        @OpenApiContentProperty(name = "form-element", type = "integer"), // random element in form-data
                        @OpenApiContentProperty(name = "file-name", isArray = true, type = "string", format = "base64") // multi-file upload
                })
        }
),
```

results in:

```json
"requestBody" : {
  "content" : {
    "text/plain" : {
      "schema" : {
        "type" : "string"
      }
    },
    "application/json" : {
      "schema" : {
        "$ref" : "#/components/schemas/LombokEntity"
      }
    },
    "image/png" : {
      "schema" : {
        "type" : "string",
        "format" : "base64"
      }
    },
    "multipart/form-data" : {
      "schema" : {
        "type" : "object",
        "properties" : {
          "form-element" : {
            "type" : "integer"
          },
          "file-name" : {
            "type" : "array",
            "items" : {
              "type" : "string",
              "format" : "base64"
            }
          }
        }
      }
    }
  },
  "required" : false
},
```

As always, snapshot version with those changes is already in repo :)
